openvdb/util/PagedArray.h

```diff
@@ -234,7 +239,7 @@ class PagedArray
     /// @param value value to be added to this PagedArray
     ///
     /// @details Constant time complexity. May allocate a new page.
-    size_t push_back(const ValueType& value)
+    inline size_t push_back(const ValueType& value)
```

Comment:
Inline qualifiers are redundant for member functions defined inside the class declaration.

Reply:
I'm finding conflicting information on this subject matter. For instance, one of the very first google search results on "C++ inline keyword" includes and a example of an application identical to mine (see code in http://www.cplusplus.com/articles/2LywvCM9/). Other forum discussions seem to suggest that the C++ standard does indeed guarantee that member functions that are defined in the declaration will be treated as inline functions. I'm happy to make the change but I'd feel much better is there was definitive proof that it is indeed redundant. Can you point me to something that settles this?

Reply:
cplusplus.com is notoriously unreliable; use cppreference.com instead.

Here's the cppreference entry on inline functions: https://en.cppreference.com/w/cpp/language/inline.

And here's the C++ standard: http://open-std.org/JTC1/SC22/WG21/docs/papers/2019/n4800.pdf (see line 4 of section 9.1.6, "The inline specifier").

Reply:
cool - second sentence in https://en.cppreference.com/w/cpp/language/inline settled it!

I've already made the code changes